src/main/flight/pid.h

```diff
@@ -390,7 +390,11 @@ typedef struct pidRuntime_s {
     float feedforwardJitterFactor;
     float feedforwardBoostFactor;
 #endif
-} pidRuntime_t;
+
+#ifdef USE_ACC
+    pt3Filter_t attitudeFilter[2];  // Only for ROLL and PITCH
+#endif
+} __attribute__((aligned)) pidRuntime_t;
```

Comment:
Why?

Reply:
I have tried again and there is no benefit or difference even in assembly code. I will remove it.

Reply:
Implicit alignment of whole `struct` is the same as largest member (If i read the specs correctly). So explicit align did nothing ...

Reply:
Yes, agree.